pkg/framework/plugins/nodeutilization/nodeutilization.go

```diff
@@ -280,14 +283,30 @@ func evictPods(
 	continueEviction continueEvictionCond,
 ) {
 
+	var includedNamespaces, excludedNamespaces sets.String
+	if evictionNamespaces != nil {
+		includedNamespaces = sets.NewString(evictionNamespaces.Include...)
+		excludedNamespaces = sets.NewString(evictionNamespaces.Exclude...)
+	}
+
 	if continueEviction(nodeInfo, totalAvailableUsage) {
 		for _, pod := range inputPods {
 			if !utils.PodToleratesTaints(pod, taintsOfLowNodes) {
 				klog.V(3).InfoS("Skipping eviction for pod, doesn't tolerate node taint", "pod", klog.KObj(pod))
 				continue
 			}
 
-			if podEvictor.PreEvictionFilter(pod) {
+			preEvictionFilterWithOptions, err := podutil.NewOptions().
+				WithFilter(podEvictor.PreEvictionFilter).
+				WithNamespaces(includedNamespaces).
+				WithoutNamespaces(excludedNamespaces).
+				BuildFilterFunc()
```

Comment:
Do we want to use these, or do we want to throw this functionality into the PreEvictionFilter directly?

Reply:
Into the PreEvictionFilter directly

Reply:
The namespace is a parameter of the strategies so it needs to stay outside of PreEvictionFilter extension point